src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -357,7 +371,7 @@ const PersonalDetails = (() => {
                     $.scrollTo($('h1#heading'), 500, { offset: -10 });
                     $(form_id).setVisibility(0);
                     $('#missing_details_notice').setVisibility(0);
-                    $('.rowCustomerSupport').setVisibility(0);
+                    $(row_customer_support).setVisibility(0);
```

Comment:
Nit: we could stick to the previous version too since we just access this variable once. It's not an issue but if we directly use classname here, we reduce the code by one line.

Reply:
@akmal-binary Agree) Done.